packages/terra-modal/src/ModalContent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import FocusTrap from 'focus-trap-react';
+import ModalOverlay from './ModalOverlay';
+
+const propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  classNameModal: PropTypes.string,
+  classNameOverlay: PropTypes.string,
+  closeOnOutsideClick: PropTypes.bool,
+  closePortal: PropTypes.func,
+  role: PropTypes.string,
+};
+
+const defaultProps = {
+  ariaLabel: null,
+  children: null,
+  classNameModal: null,
+  classNameOverlay: null,
+  closeOnOutsideClick: true,
+  role: 'dialog',
+};
+
+
+class ModalContent extends React.Component {
+
+  componentDidMount() {
+    // Disable background scrolling
+    document.body.style.overflow = 'hidden';
```

Comment:
Would the Overlay prevent any scroll events? Many Orion applications utilize a SlidePanel as a base component that manages its' own overflow, without the overlay I think these would still scroll. I think the current implementation has the full screen overlay prevent any scrolling.